### PR TITLE
sci-libs/openblas: Version Bump to 0.2.18

### PR DESCRIPTION
**Pull request by CraigInches**
New Ebuild
Changes to patches to remove paths

### CODE REVIEW

**Comment by CraigInches** [comment]
Will Fail to build until the patch used by all the ebuilds is updated to remove the unneeded contributors additions

File: sci-libs/openblas/files/openblas-0.2.11-cpuid_x86.patch [hunk 1]
```diff
@@ -1,5 +1,5 @@
---- openblas-9999-orig/cpuid_x86.c	2014-02-23 15:25:22.199394761 +1100
-+++ openblas-9999/cpuid_x86.c	2014-02-23 16:41:02.727372869 +1100
+--- cpuid_x86.c	2014-02-23 15:25:22.199394761 +1100
++++ cpuid_x86.c	2014-02-23 16:41:02.727372869 +1100
```
Comment (jlec) [inline sci-libs/openblas/files/openblas-0.2.11-cpuid_x86.patch]: Are you sure that works? ```eapply```will always use ```-p1```which strips the first bit.

Comment (CraigInches) [inline sci-libs/openblas/files/openblas-0.2.11-cpuid_x86.patch]: yeh, it failed with -p1 for me, but worked with the removal.

**Comment by heroxbd** [comment]
Obsolete, see PR #800.

**Pull request closed without merging**